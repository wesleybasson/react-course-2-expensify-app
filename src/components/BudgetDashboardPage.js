import React from 'react';
import BudgetSummary from './BudgetSummary';

const BudgetDashboardPage = () => (
    <div>
        <BudgetSummary />
    </div>
);

export default BudgetDashboardPage;