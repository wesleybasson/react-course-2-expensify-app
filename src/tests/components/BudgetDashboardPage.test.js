import React from 'react';
import { shallow } from 'enzyme';
import BudgetDashboardPage from '../../components/BudgetDashboardPage';

test('should render BudgetDashboardPage correctly', () => {
    const wrapper = shallow(<BudgetDashboardPage />)
    expect(wrapper).toMatchSnapshot()
})