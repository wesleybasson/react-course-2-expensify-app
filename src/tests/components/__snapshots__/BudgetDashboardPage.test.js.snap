// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render BudgetDashboardPage correctly 1`] = `
<div>
  <Connect(BudgetSummary) />
</div>
`;
